appsec/embed.go

```diff
@@ -5,10 +5,16 @@
 
 package appsec
 
-import _ "embed" // Blank import comment for golint compliance
+import (
+	_ "embed" // Blank import comment for golint compliance
+	"unsafe"
+)
 
-// StaticRecommendedRules holds the recommended AppSec security rules (v1.14.2)
-// Source: https://github.com/DataDog/appsec-event-rules/blob/1.14.2/build/recommended.json
-//
-//go:embed rules.json
-var StaticRecommendedRules string
+var (
+	//go:embed rules.json
+	staticRecommendedRules []byte
+
+	// StaticRecommendedRules holds the recommended AppSec security rules (v1.14.2)
+	// Source: https://github.com/DataDog/appsec-event-rules/blob/1.14.2/build/recommended.json
+	StaticRecommendedRules = unsafe.String(&staticRecommendedRules[0], len(staticRecommendedRules))
```

Comment:
This is to retain the exported symbol (otherwise it's a breaking change) while avoiding to cause a full copy of the backing data to be done (because it's unnecessary).

That symbol would be dropped in a future version.